quizapp/src/main/java/billennium/quizapp/repository/UserRepository.java

```diff
@@ -0,0 +1,11 @@
+package billennium.quizapp.repository;
+
+import billennium.quizapp.entity.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.Optional;
+
+public interface UserRepository extends JpaRepository<User, Long> {
+    
+    Optional<User> findById(Long aLong);
```

Comment:
`aLong` ? wg mnie powinno być `id`

Reply:
moj intellij sam tak dodaje jkabym chciał overide tej metody, wywale to wcale :)

Reply:
wgl potrzbene jest definowanie tej metody?  Ona nie jets domyslna optional ?